policy/modules/system/systemd.fc

```diff
@@ -63,6 +70,8 @@
 
 /run/systemd/ask-password(/.*)?	gen_context(system_u:object_r:systemd_passwd_runtime_t,s0)
 /run/systemd/ask-password-block(/.*)?	gen_context(system_u:object_r:systemd_passwd_runtime_t,s0)
+/run/systemd/generator(/.*)?	gen_context(system_u:object_r:systemd_generator_runtime_t,s0)
+/run/systemd/generator\.late(/.*)?	gen_context(system_u:object_r:systemd_generator_runtime_t,s0)
```

Comment:
I agree with doverride, this should be revised as unit locations.

Reply:
Ok, I will leave the changes for systemd_generator_runtime_t out and they can be considered in a future patch.  For now the type for /run/systemd/generator/.* will remain init_runtime_t.

I will have an updated squashed patch shortly.

Reply:
i recorded some of my thoughts on systemd units: https://www.youtube.com/watch?v=FbZfe80usJI